docs/quick-start.md

```diff
@@ -89,7 +96,7 @@ isbsvc-default-js-2                3/3     Running   0          10m
 # Port-forward the HTTP endpoint so we can post data from the laptop
 kubectl port-forward even-odd-in-0-xxxx 8444:8443
 
-# Post data to the HTTP endpoint
+# Open a new terminal tab to post data to the HTTP endpoint
```

Comment:
Feel like a simple `Post xxxx` makes more sense, more detail like `open a terminal` might confuse ppl, as lots of ppl do not use it, and even have no idea about it.

Reply:
Make sense, will address.